<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiliTest</class>
 <widget class="QWidget" name="BiliTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="urlInEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="bvidButton">
       <property name="text">
        <string>Form URL to BVID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bvidOutEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bvidInEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="cidButton">
       <property name="text">
        <string>From BVID to CID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cidOutEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>CID In:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlCidEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>BVID In:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlBvidEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="videoButton">
       <property name="text">
        <string>GetVideo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>CID In:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="danEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="danButton">
       <property name="text">
        <string>Fetch danmaku</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="searchEdit"/>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="bigOutEdit"/>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="saveDanButton">
     <property name="text">
      <string>Save Danmaku</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
